uaclient/clouds/identity.py

```diff
@@ -19,7 +19,7 @@
 
 
 # Mapping of datasource names to cloud-id responses. Trusty compat with Xenial+
-DATASOURCE_TO_CLOUD_ID = {"azurenet": "azure", "ec2": "aws"}
+DATASOURCE_TO_CLOUD_ID = {"azurenet": "azure", "ec2": "aws", "gce": "gce"}
```

Comment:
This dict translates cloud-init to ua-client so it should be "gce" -> "gcp". Ultimately it's only needed if we release Gcp trusty images.  So either we drop this is we don't plan trusty pro image support, or we change it to the following:

```suggestion
DATASOURCE_TO_CLOUD_ID = {"azurenet": "azure", "ec2": "aws", "gce": "gcp"}
```